dhcpv6/option_prefixdelegation.go

```diff
@@ -0,0 +1,72 @@
+package dhcpv6
```

Comment:
Is this file coming from a bad rebase?

Reply:
yes, it does look like it for both files. Looking at the history they have been renamed but for some reason during rebase it just decided to keep them both.. no idea why as I didn't go interactive with it.
Anyway I removed them both now.